### ISSUE TITLE
babel generated code getting picked up by labs coverage reporter

### ISSUE THREAD

**Issue by AKST** [nlf/lab-babel#5]
### The issue

Babel generates some code that acts like polyfill for handling class creation, class sub classing, modules, and whatever else **[1]**. This code is getting picked up in the coverage report. I tried wrapping the top section of the file in the coverage flags **[2]**, but it gets thrown out of whack by the inserted `"use strict"; expression at the top, and it's own flags.

```
// $lab:coverage:on$
"use strict";
// $lab:coverage:off$
```
### Possible solutions?
1.  Is it possible to wrap this code in the lab coverage comment flags? **[2]** 
2. Alternatively are source maps able to provide enough information to the coverage checker to ignore certain lines? 
### Replication

You should be able to replicate this with the following relevant npm package versions.
- `lab` **@** `5.13.0`
- `lab-babel` **@** `^1.0.1`

Alternatively [this project](https://github.com/AKST/akst.io-backend/tree/926f734a38bb8ca60ffd8c3a49b354fbf82e15b2) replicates the problem & outputs the a html test report with coverage examples to `./dist/lab.html` after running `npm test`. Check `src/util/route-commons.js` for an example of this problem.

---

**[1]:**  The code will generally look like this, and will be at the top of file. 

```
var _slicedToArray = (function () { function sliceIterator(arr, i) { var _arr = []; var _n = true; var _d = false; var _e = undefined; try { for (var _i = arr[Symbol.iterator](), _s; !(_n = (_s = _i.next()).done); _n = true) { _arr.push(_s.value); if (i && _arr.length === i) break; } } catch (err) { _d = true; _e = err; } finally { try { if (!_n && _i["return"]) _i["return"](); } finally { if (_d) throw _e; } } return _arr; } return function (arr, i) { if (Array.isArray(arr)) { return arr; } else if (Symbol.iterator in Object(arr)) { return sliceIterator(arr, i); } else { throw new TypeError("Invalid attempt to destructure non-iterable instance"); } }; })();
function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } else { var newObj = {}; if (obj != null) { for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) newObj[key] = obj[key]; } } newObj["default"] = obj; return newObj; } }
```

**[2]:**  `$lab:coverage:on$` and `$lab:coverage:off$`

**Comment by AKST** [comment]
**Update**: I've ended up switching to my own transformation file because the transformation was more specific than just vanilla `bable-core`. I tested coverage again and for whatever reason I no longer have the problem where the polyfill-esque was fudging the coverage.

The transformation file is basically a mildly adjusted version of the babel.js example on the `lab` Github page, but the example as is will work fine.
